torch/_inductor/graph.py

```diff
@@ -998,6 +1047,10 @@ def debug(msg):
                         stride_order = ir.NHWC_STRIDE_ORDER
                     result = ir.ExternKernel.require_stride_order(result, stride_order)
 
+                if is_output and len(strides):
```

Comment:
Does this need to be done for non user visible outputs?

Reply:
Yes, because it interacts with aot_autograd